cves/CVE-2018-13304.yml

```diff
@@ -135,7 +138,13 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    This vulnerability was caught shortly after the support for 8-bit 
+    and Simple Studio Profile where added, and a few other bugs/vulnerabilities
+    where fixed before this specific problem was addressed. The first few fixes
+    aimed at standardizing the profile check, while this commit addressed one 
+    that was missed. Around this time, the profile tracking was being refactored,
+    which made it even easier to be caught so quickly. 
```

Comment:
Good job describing the timeline. This team is lucky they caught this while this feature was already being refactored.